<ui version="4.0" >
 <class>ListDialog</class>
 <widget class="QDialog" name="ListDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cargar Modelo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item rowspan="4" row="0" column="0" >
      <widget class="QListWidget" name="list" />
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>&amp;Agregar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="editButton" >
       <property name="text" >
        <string>&amp;Editar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="deleteButton" >
       <property name="text" >
        <string>E&amp;liminar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="velocityButton" >
       <property name="text" >
        <string>Velocidades</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>156</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>156</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
